### DISCUSSION TITLE
How to create additional database in container?

### DISCUSSION THREAD

**Discussion by dkern** [wardenenv/warden#722] [Help & Support]
Hello!

I need to be able to create a blank database beside the "magento" database in the container, because I have a project what needs to access a second database to pull some data from.

I tried adding a second database with DBeaver. But when creating I get the issue:

> SQL-Error [1044] [42000]: (conn=9) Access denied for user 'magento'@'%' to database 'test'
(conn=9) Access denied for user 'magento'@'%' to database 'test'
(conn=9) Access denied for user 'magento'@'%' to database 'test'

Is there any other user with more privileges or another way to create this db?

Thanks!

**Comment by navarr** [comment]
Yes! There is a `root` user with the same password.

**Reply by dkern** [reply]
Ahh, thank you. That worked!

That would be a good info for the documentation too. ;)
